src/trade/bid/bid.resolver.ts

```diff
@@ -48,11 +61,38 @@ export class BidResolver {
             auction: { connect: { id: auctionId } },
             bitPrice,
         }
+        const isExist = await this.bidsService.getBidByUserId(userId)
```

Comment:
`isExist` should be boolean type. `!!` - can convert to boalean or rename variable.